posthog/test/base.py

```diff
@@ -692,3 +692,20 @@ def wrapped(self, *args, **kwargs):
                 self.assertQueryMatchesSnapshot(query)
 
     return wrapped
+
+
+def also_test_with_different_timezone(fn):
+    def fn_with_different_timezone(self, *args, **kwargs):
```

Comment:
Problem: When we call frozen_time and/or pass timestamps for events within these tests these will be in UTC, causing results to change.

Reply:
Yeah I think the mechanism isn't great and I'll think more on it later. Currently can't just slap the decorator everywhere and hope it works, need to be a bit selective.

What it should solve for right away though on the few tests I've added it to is to prevent the case we have now in prod where we get all 0s for a graph